sources/api/host-containers/README.md

```diff
@@ -4,9 +4,13 @@ Current version: 0.1.0
 
 ## Background
 
-host-containers is a tool that queries the API for the currently enabled host containers and
-ensures the relevant systemd service is enabled/started or disabled/stopped for each one depending
-on its 'enabled' flag.
+host-containers ensures that host containers are running as defined in system settings.
+
+It queries the API for their settings, then configures the system by:
+* if defined, creating a user-data file in the host container's persistent storage area
```

Comment:
This reads a little weird.  Perhaps the bullet point starts with `creating a user-data file....` and ends with `if defined in settings` or something like that